internal_transfers/actions/tests/e2e/pipeline.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { TenderlySimulator } from "../../src/simulate/tenderly";
+import { internalizedTokenImbalance } from "../../src/pipeline";
+import { getDB } from "../../src/database";
+import { getTxData } from "./helper";
+
+const { TENDERLY_USER, TENDERLY_PROJECT, TENDERLY_ACCESS_KEY } = process.env;
+
+const simulator = new TenderlySimulator(
+  TENDERLY_USER || "INVALID_USER",
+  TENDERLY_PROJECT || "TENDERLY_PROJECT",
+  TENDERLY_ACCESS_KEY || "TENDERLY_ACCESS_KEY"
+);
+
+const db = getDB("postgresql://postgres:postgres@localhost:5432/postgres");
```

Comment:
I thought we removed this in an earlier [PR](https://github.com/cowprotocol/solver-rewards/pull/268), no?